include/cadmium/celldevs/coupled/cells_coupled.hpp

```diff
@@ -0,0 +1,133 @@
+         */
+        explicit cells_coupled(std::string const &id) : cadmium::dynamic::modeling::coupled<T>(id), vicinities_() {}
+
+        /**
+         * @brief adds a single Cell-DEVS cell to the coupled model
+         * @tparam CELL_MODEL model type of the cell to be included
+         * @tparam Args any additional parameter required for initializing the cell model
+         * @param cell_id ID of the cell. It must not be already defined in the coupled model
+         * @param initial_state Initial state of the cell
+         * @param vicinities unordered map {neighboring cell ID: vicinity kind}
+         * @param delayer_id ID identifying the type of output delayer buffer used by the cell.
+         * @param args any additional parameter required for initializing the cell model
+         */
+        template <template <typename> typename CELL_MODEL, typename... Args>
+        void add_cell(C const &cell_id, S const &initial_state, CV const &vicinities,
+                      std::string const &delayer_id, Args&&... args) {
+            add_cell_vicinity(cell_id, vicinities);
+            cadmium::dynamic::modeling::coupled<T>::_models.push_back(
+                    cadmium::dynamic::translate::make_dynamic_atomic_model<CELL_MODEL, T>(get_cell_name(cell_id),
+                                                                                          cell_id, initial_state,
+                                                                                          vicinities, delayer_id,
+                                                                                          std::forward<Args>(args)...));
+        }
+
+        /**
+         * adds a single Cell-DEVS cell to the coupled model. Vicinity is set to its default value
+         * @tparam CELL_MODEL model type of the cell to be included
+         * @tparam Args any additional parameter required for initializing the cell model
+         * @param cell_id ID of the cell. It must not be already defined in the coupled model
+         * @param initial_state Initial state of the cell
+         * @param neighbors vector containing all the IDs of neighboring cells
+         * @param delayer_id ID identifying the type of output delayer buffer used by the cell.
+         * @param args any additional parameter required for initializing the cell model
+         */
+        template <template <typename> typename CELL_MODEL, typename... Args>
+        void add_cell(C const &cell_id, S const &initial_state, std::vector<C> const &neighbors,
+                      std::string const &delayer_id, Args&&... args) {
+            CV vicinity = CV();
+            for (auto const &neighbor: neighbors) {
+                vicinity.insert({neighbor, V()});
+            }
+            add_cell<CELL_MODEL, Args...>(cell_id, initial_state, vicinity, delayer_id, std::forward<Args>(args)...);
+        }
+
+        /// The user must call this method right after having included all the cells of the scenario
```

Comment:
/// -> //

Reply:
Same as above. I also use /// for documenting small methods with no arguments nor return types.